pkg/mcs/tso/server/server.go

```diff
@@ -324,12 +328,12 @@ func (s *Server) AddStartCallback(callbacks ...func()) {
 // IsServing implements basicserver. It returns whether the server is the leader
 // if there is embedded etcd, or the primary otherwise.
 func (s *Server) IsServing() bool {
-	return s.participant.IsLeader() && atomic.LoadInt64(&s.isServing) == 1
+	return atomic.LoadInt64(&s.isServing) == 1 && s.participant.IsLeader()
 }
 
-// GetPrimary returns the primary provider of this tso server.
-func (s *Server) GetPrimary() bs.MemberProvider {
-	return s.participant.GetLeader()
+// GetLeaderListenUrls gets service endpoints from the leader in election group.
```

Comment:
Do we need also to add comment for index 0?

Reply:
added "// The entry at the index 0 is the primary's service endpoint."